api/src/main/java/net/md_5/bungee/api/event/ResourcePackResponseEvent.java

```diff
@@ -0,0 +1,66 @@
+package net.md_5.bungee.api.event;
+
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.ToString;
+import net.md_5.bungee.api.connection.Connection;
+import net.md_5.bungee.api.connection.ProxiedPlayer;
+
+/**
+ * Called when a player takes action on a resource pack request
+ * sent to their client by the proxy or server.
+ */
+@Getter
+@ToString(callSuper = false)
+@EqualsAndHashCode(callSuper = false)
+public class ResourcePackResponseEvent extends TargetedEvent
+{
+
+    /**
+     * Player sending the response.
+     */
+    private final ProxiedPlayer player;
```

Comment:
Uh I don't think other targeted events have this. Isn't player always just going to be sender?

Reply:
yup you're right i'm gonna transform this into a basic event

Reply:
I think it should've remained a targeted event?
Every other event, eg, TabCompleteResponse is like that